src/pytorch_lightning/cli.py

```diff
@@ -485,7 +487,7 @@ def _prepare_subcommand_parser(self, klass: Type, subcommand: str, **kwargs: Any
         parser = self.init_parser(**kwargs)
         self._add_arguments(parser)
         # subcommand arguments
-        skip = self.subcommands()[subcommand]
+        skip: Set[Union[str, int]] = set(self.subcommands()[subcommand])
```

Comment:
Doesn't `self.subcommands()[subcommand]` already return a set?

Reply:
If `set` is removed, mypy complains because the type is not `Set[Union[str, int]]`.